docs/contributors/outreach.md

```diff
@@ -13,15 +67,21 @@ https://www.meetup.com/WordPress-Melbourne/events/241543639
 
 https://wpmeetups.de/termin/29-wp-meetup-stuttgart-gutenberg-editor-rueckblick-wordcamp-europe/
 
-## Testing events: 
-https://www.meetup.com/Turku-WordPress-Meetup/events/241195076/
-
-https://www.meetup.com/Vancouver-WordPress-Meetup-Group/events/241575161/
-
-## Calls for testing: 
-https://make.wordpress.org/test/2017/06/27/call-for-testing-gutenberg/
 
-http://www.wpswfl.org/new-wordpress-editor-gutenberg-early-beta-needs-testers/
+## Meetups
```

Comment:
I think we should remove the meetups from the handbook. There were hundreds of meetups about Gutenberg. The links to meetup.com are nor really helpful. Articles and Videos are more valuable.

Reply:
Agreed. There were a lot of old material I wasn't sure what to do with.

Reply:
Removed in https://github.com/WordPress/gutenberg/pull/13352/commits/d77219986fc7cc2dfce4fd805a682c930a2d327c